_CoqProject

```diff
@@ -1,4 +1,4 @@
--Q theories Actuary
+-R theories Actuary
```

Comment:
```suggestion
-Q theories Actuary
```
I recommend having `-Q` here, since this is stronger than `-R` (you always have to `Require` modules by their "full" name, which is good practice).